index.html

```diff
@@ -422,7 +422,119 @@ <h1 class="contact_form_heading">
 
   </div>
   
+  <div class="modal_mob">
```

Comment:
[ OPTIONAL ] Pop-up buttons are overflowing down on the mobile view. Moreover, icons in buttons are not aligned horizontally with text. Kindly see the screenshot below and fix the issues.
![popup-mobile](https://user-images.githubusercontent.com/35504149/132833828-ff7e0fe3-2c75-40e9-94a2-4f2adc40625a.png)